addons/web/static/tests/views/pivot_tests.js

```diff
@@ -1956,6 +1956,38 @@ QUnit.module('Views', {
         actionManager.destroy();
     });
 
+    QUnit.test('Click on the divider in the menu "Measures" does not crash', async function (assert) {
+        assert.expect(1);
+
+        const pivot = await createView({
+            View: PivotView,
+            model: "partner",
+            data: this.data,
+            arch: `<pivot/>`
+        });
+        await testUtils.dom.click(pivot.$buttons[0].querySelector('button'));
+        await testUtils.dom.click(pivot.$buttons[0].querySelector('.o_pivot_measures_list div.dropdown-divider'));
+        assert.ok(true);
```

Comment:
I would check that it is still open instead

Reply:
Does it fail all the time without the patch? I struggled to reproduce the issue if I remember correctly.

Reply:
> Does it fail all the time without the patch? I struggled to reproduce the issue if I remember correctly.

I would say it regularly happens.